pkg/cephfs/volume.go

```diff
@@ -17,27 +17,17 @@ limitations under the License.
 package cephfs
 
 import (
-	"fmt"
-	"os"
 	"path"
 
-	"k8s.io/klog"
+	"github.com/ceph/ceph-csi/pkg/util"
 )
 
 const (
 	cephVolumesRoot = "csi-volumes"
```

Comment:
Doesn't this need to be changed?

Reply:
+1

But, some further comments,
- @poornimag For 1.0 volumes the volume root would need to be retained as "csi-volumes", as these are not created by the `ceph fs subvolume ...` commands
- For post 1.0 volumes, the volumes root **maybe** `/volume/<subvolume-group>/<csi-vol-id>`
  - can this path be assumed as the volume root path? @ajarr 
  - There was a prior discussion that we would be able to fetch this root path from one of the subcommands, to aid in keeping the interface abstraction in place (during node plugin actions). IOW, do not leak the internals of exactly where the subvolumes are created, but pass it back when asked for. I would prefer that, to hard coding it in the code here, so that any changes can be easily accommodated. @ajarr ?

Reply:
So you can fetch the absolute path of the subvolume,
`ceph fs subvolume getpath <vol name/fs name> <subvol name/csi-vol-id> --group_name csi` 
The command would return, `/volumes/csi/<csi-vol-id>`.

@ShyamsundarR , does that answer your question?

Reply:
Yes, thanks. This helps in the nodeplugin to not have to assume the path, and change/use something like `cephVolumesRoot` variable.

@poornimag this needs to be incorporated in the publish sequence to fetch the path rather than construct it.

Reply:
Done